### PR TITLE
Rename remaining EIP4844 references to Deneb

### PR DESCRIPTION
**Pull request by StefanBratanov**
## PR Description

- Now that reference tests are updated and deneb renaming is part of the specs, can remove the remaining references to EIP-4844
- Removed the TODO from `BlobsSidecarsByRangeMessageHandler` and `FetchBlockAndBlobsSidecarTask` because these classes will be removed when implementing the blob decoupling

## Fixed Issue(s)
N/A

## Documentation

- [X] I thought about documentation and added the `doc-change-required` label to this PR if updates are required.

## Changelog

- [X] I thought about adding a changelog entry, and added one if I deemed necessary.

### CODE REVIEW

File: networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/rpc/beaconchain/methods/BlobsSidecarsByRangeMessageHandler.java [hunk 1]
```diff
@@ -131,9 +131,7 @@ public void onIncomingMessage(
   }
 
   /**
-   * TODO: Update link for Deneb
-   *
-   * <p>See <a
+   * See <a
    * href="https://github.com/ethereum/consensus-specs/blob/dev/specs/eip4844/p2p-interface.md#blobssidecarsbyrange-v1">BlobsSidecarsByRange
```
Comment (tbenr) [inline networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/rpc/beaconchain/methods/BlobsSidecarsByRangeMessageHandler.java:135]: you haven't updated the link here

Comment (StefanBratanov) [reply networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/rpc/beaconchain/methods/BlobsSidecarsByRangeMessageHandler.java:135]: that would be deleted and there is no reference to it now with the new spec, so just decided to leave it there

Comment (tbenr) [reply networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/rpc/beaconchain/methods/BlobsSidecarsByRangeMessageHandler.java:135]: oh but it is the pre-decoupling one. Never mind.

**Review by tbenr — APPROVED** [review]
LGTM

**Pull request merged**